constraints.pro

```diff
@@ -0,0 +1,67 @@
+% "name" is required for all workspaces (including the root).
+\+ gen_enforced_field(WorkspaceCwd, 'name', null).
+
+% "description" is required for all workspaces.
+\+ gen_enforced_field(WorkspaceCwd, 'description', null).
+
+% "repository.type" must be "git" for all workspaces (including the root).
+gen_enforced_field(WorkspaceCwd, 'repository.type', 'git').
+
+% "repository.type" must start with "git@github.com:mcmire/" for all workspaces
```

Comment:
Need to either figure this out or remove this.

Reply:
Fixed in [5f6205c](https://github.com/MetaMask/controllers/pull/831/commits/5f6205c0cf8b82361e1169af5eb8f189fdaa65c8).